apps/rush/src/utilities/TaskManager.ts

```diff
@@ -0,0 +1,156 @@
+import {
+  RushConfiguration,
+  RushConfigurationProject,
+  IRushLinkJson
+} from '@microsoft/rush-lib';
+import { JsonFile } from '@microsoft/node-core-library';
+
+import TaskRunner from '../taskRunner/TaskRunner';
+import ProjectBuildTask from '../taskRunner/ProjectBuildTask';
+
+export class TaskManager {
```

Comment:
Can you add tests for this class?

Reply:
Yeah, why not

---
In reply to: [145581577](https://github.com/Microsoft/web-build-tools/pull/387#discussion_r145581577) [](ancestors = 145581577)